packages/shared/src/components/auth/ActiveUsersCounter.tsx

```diff
@@ -11,11 +11,15 @@ export function ActiveUsersCounter(): ReactElement {
     getRandomUsersCount,
   );
 
+  if (!data) {
+    return null;
+  }
+
   return (
     <div className="relative mb-2 flex w-fit flex-row items-center rounded-8 bg-surface-float p-1 pr-3">
       <UserIcon secondary size={IconSize.Small} />
       <span className="ml-3 typo-footnote">
-        <strong>{data} </strong>
+        <strong>{data.toLocaleString()} </strong>
```

Comment:
Keep in mind this will also do some locale [specific conversions which are more ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#using_locales)then just fraction separators.
<img width="651" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/31c36067-7288-47b7-bc1e-ca3c183697c8">

Maybe specifying `locales` in order would be better to avoid it?

Reply:
Oh, very cool. Didn't know about this. Let me spend more time on how we can manipulate better.

Reply:
Applied `en-US` as locale as part of the [requirement](https://dailydotdev.slack.com/archives/C06K5KAHM0X/p1710100529461399?thread_ts=1709785127.865909&cid=C06K5KAHM0X) to use comma. We currently don't support different languages, so using the current local wouldn't make sense.